web/impact/impact/tests/test_judging_round_list_view.py

```diff
@@ -149,6 +149,13 @@ def test_ignore_clearance_request_param_works(self):
             round_ids = [item["id"] for item in results]
             self.assertTrue(the_round.id in round_ids)
 
+    def test_staff_user_can_view(self):
+        email = self.staff_user().email
+        with self.login(email=email):
+            url = reverse(JudgingRoundListView.view_name)
+            response = self.client.get(url)
+            self.assertEqual(response.status_code, 200)
```

Comment:
Since the V1 permissions grant staff access to impact views, they can access this view but to actually see the results they have to be global managers as specified here: https://github.com/masschallenge/impact-api/blob/560dce5c8ad3cc293ee0e3a38ce2305607bf2d0a/web/impact/impact/v1/views/judging_round_list_view.py#L34. So I'm refactoring the test to reflect this